pdns/recursordist/docs/settings.rst

```diff
@@ -865,6 +865,43 @@ To prevent this, add a Negative Trust Anchor (NTA) for this zone in the `lua-con
 If this forwarded zone is signed, instead of adding NTA, add the DS record to the `lua-config-file`_.
 See the :doc:`dnssec` information.
 
+.. _setting-forward-zones-catalog
+
+``forward-zones-catalog``
+-------------------------
+.. versionadded:: 4.10.0
+-  'zonename=IP[:port]'
```

Comment:
```suggestion
-  List of 'zonename=IP[:port]', separared by semicolons
```

Reply:
The config parser supplies commas between items if `+=` is used, so the text will need to indicate that (as it does for other options already).